test/distributed/org/apache/cassandra/distributed/upgrade/UpgradeTestBase.java

```diff
@@ -383,7 +483,7 @@ private static void triggerGC()
     protected TestCase allUpgrades(int nodes, int... toUpgrade)
     {
         return new TestCase().nodes(nodes)
-                             .upgradesToCurrentFrom(v30)
+                             .applicableVersionsRange(OLDEST, CURRENT)
```

Comment:
changed from `v30` to `OLDEST` if this is supposed to be `allUpgrades`

It is of no consequence on v5, but it's clearer I think.